Samples/VirtualCamera/README.md

```diff
@@ -0,0 +1,93 @@
+# Virtual Camera project
+

```

Comment:
We need a big and bold section on minimum OS version needed for this to work.

Reply:
add version is TBD